src/main/resources/META-INF/plugin.xml

```diff
@@ -231,7 +231,7 @@
 
                 <action id="cody.editCodeAction"
```

Comment:
rename here too?

Reply:
Not needed, as this is not called from agent, and there is one good reason to not do renames if not needed:
I think it can mess-up shortcuts if users already assigned custom ones to their actions.